src/sentry/models/organization.py

```diff
@@ -119,7 +119,7 @@ def get_for_user(self, user, scope=None, only_visible=True):
         return [r.organization for r in results]
 
 
-@region_silo_model
+@all_silo_model
```

Comment:
I needed to change the silo requirement for organization because the `organization -> region` api requires an organization object. Until we settle on where the Org table should live, I think this shouldn't be restrictive

Reply:
@markstory The organization table is going to end up in the control, silo right?